File: corpus/py/F;0702.py
Scanner ler= new Scanner (System.in)
int x
print("Quantos elementos tem o vetor?")
x= int(input())
for i=0 in range(0, x):
    print("Quais os valores do vetor na posição : " +i)
chamado(v, x)
def chamado(int v[],int x):
    Scanner ler= new Scanner (System.in)
    int l
    print("Quantas linhas tem a matriz?")
    l= int(input())
    int c
    print("Quantas colunas tem a matriz?")
    c= int(input())
    for i=0 in range(0, l):
        for j=0 in range(0, c):
            print("Quais são os valores da matriz na linha "+ i + "e na coluna "+ j)
    return 0

File: corpus/2018q1/_p2q3_ok/_check/F;0702.java

import java.util.Scanner;

public class QUESTAO3 {

    public static void main(String[] args) {

        Scanner ler= new Scanner (System.in);
        int x;
        System.out.println("Quantos elementos tem o vetor?");
        x= ler.nextInt();
        int [] v = new int [x];
            for (int i=0; i<x; i++){
                System.out.println("Quais os valores do vetor na posição : " +i);
                v[i]= ler.nextInt();
    }

            chamado(v, x);
  }
    public static int chamado (int v[],int x){
        Scanner ler= new Scanner (System.in);
        int l;
        System.out.println("Quantas linhas tem a matriz?");
        l= ler.nextInt();
        int c;
        System.out.println("Quantas colunas tem a matriz?");
        c= ler.nextInt();
        int[][] M = new int [l][c];
         for (int i=0; i<l; i++){
             for (int j=0; j<c; j++){
                 System.out.println("Quais são os valores da matriz na linha "+ i + "e na coluna "+ j);
                 M[i][j]=ler.nextInt();
             }
         }

        return 0;

    }

}
